src/style.css

```diff
@@ -30,15 +31,14 @@ body {
 }
 
 #app {
-  background-color: var(--white-background);
+  background-color: white;
   display: flex;
   flex-direction: column;
   gap: 1rem;
   border-radius: 8px;
-  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
-  padding: 1.5rem;
-  margin: 84px 12px 0 12px;
-  width: 100%;
+  box-shadow: 0 0 20px darkgray;
```

Comment:
- [optional] please add a little color(even black) to the clear all complete so it can be more visible
- [optional] Even with the clear all complete added, the CRUD method should still work. i cant seem to delete a task unless it is marked as complete